src/blog.rs

```diff
@@ -198,10 +199,14 @@ impl Blog {
                 .expect("Escaping HTML in-memory failed");
 
             entries.push(
-                // Additionally, we could add author or category information here
+                // Additionally, we could add category or other extra information here
                 atom::Entry {
                     title: post.title.clone(),
-                    id: post_path.to_string_lossy().to_string(),
+                    id: PathBuf::from(&blog_root)
```

Comment:
Could we add a comment on why to `post_path` alone is needed?

Reply:
Done